### PR TITLE
Added yaml validation task

### PR DESCRIPTION
**Pull request by linusnorton**
### Change description ###

This change adds a gradle task that will validate all the yaml configuration files in the project

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/command/ValidateYamlCommand.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    public ValidateYamlCommand(LocalConfigurationLoader loader) {
+        this.loader = loader;
+    }
+
+    public boolean run() {
+        URL url = Thread
+            .currentThread()
+            .getContextClassLoader()
+            .getResource("bundleconfiguration/");
```
Comment (JeroenRijks) [inline src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/command/ValidateYamlCommand.java]: will this always work? what if bundleconfiguration hasn't been assigned a value?

File: src/test/java/uk/gov/hmcts/reform/em/orchestrator/endpoint/CcdCloneBundleControllerTest.java [hunk 2]
```diff
@@ -30,7 +31,7 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @RunWith(SpringRunner.class)
-@SpringBootTest
+@SpringBootTest(classes = {Application.class})
```
Comment (JeroenRijks) [inline src/test/java/uk/gov/hmcts/reform/em/orchestrator/endpoint/CcdCloneBundleControllerTest.java]: ??

Comment (linusnorton) [reply src/test/java/uk/gov/hmcts/reform/em/orchestrator/endpoint/CcdCloneBundleControllerTest.java]: The main package now has multiple `@SpringBootApplication` classes. The unit tests need to know which one to boot in order to run the tests.

File: src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/command/ValidateYamlCommand.java (hunk 1 above)
Comment (linusnorton) [reply src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/command/ValidateYamlCommand.java]: it's referencing the folder inside `main/resources/`

Comment (JeroenRijks) [reply src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/command/ValidateYamlCommand.java]: ah cool. There are multiple - which one is it gonna look at?

File: src/test/java/uk/gov/hmcts/reform/em/orchestrator/endpoint/CcdCloneBundleControllerTest.java (hunk 2 above)
Comment (JeroenRijks) [reply src/test/java/uk/gov/hmcts/reform/em/orchestrator/endpoint/CcdCloneBundleControllerTest.java]: makes sense, the CliApplication class :+1:

File: src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/command/ValidateYamlCommand.java (hunk 1 above)
Comment (linusnorton) [reply src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/command/ValidateYamlCommand.java]: all of them, it prints something like:

```
2019-07-10T14:23:26.574 INFO  [main] u.g.h.r.e.o.a.command.ValidateYamlCommand Validating example.yaml succeeded
2019-07-10T14:23:26.575 INFO  [main] u.g.h.r.e.o.a.command.ValidateYamlCommand Validating example-with-filename.yaml succeeded
2019-07-10T14:23:26.575 INFO  [main] u.g.h.r.e.o.a.command.ValidateYamlCommand 0 error(s) found```

**Pull request closed without merging**